### PR TITLE
[JA] Add Do you really want to ask peppy

### PR DESCRIPTION
**Pull request by ethernetcat**
Add Japanese translation of `Do_you_really_want_to_ask_peppy`.

### CODE REVIEW

File: wiki/Do_you_really_want_to_ask_peppy/ja.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+こんにちは。個人的に私（[peppy](https://osu.ppy.sh/users/2)）に連絡したいというあなたの衝動は理解していますが、これはあなたが私に助け/サポートを求る場所**ではない**ことを最初に理解するようにお願いします。私は個人的なメッセージとは別にosu!のサポートを継続して行っています。また、フォーラムのPMに回答するのは非常に難しいです。そのため、次のいずれかで連絡をしてください。
+
+## テクニカルサポート
+
+ほとんどの技術的な問題は、[ヘルプフォーラム](https://osu.ppy.sh/community/forums/5)で検索または投稿することで解決できます。問題を解決できる人が他にもたくさんいるので、私に連絡するよりも早く解決策を見つけることができるでしょう。
+
+## 一般的なosu!サポート
+
+アカウントの問題、支払いの問題、BAN、トーナメントに関するお問い合わせ、及び**「osu!」に関するヘルプが必要なもの**については[accounts@ppy.sh](mailto:accounts@ppy.sh)（アカウントクエリ）または、[osu@ppy.sh](mailto:osu@ppy.sh)（何でも）にメールを送信してください。サポートメンバーがオンラインの場合、表示しているページの右下にある小さな[?]ボタンを使用することもできます。
```
Comment (cl8n) [inline wiki/Do_you_really_want_to_ask_peppy/ja.md]: ここにかぎ括弧は必要ないと思う。さらに最後の文はenで削除された
```suggestion
アカウントの問題、支払いの問題、BAN、トーナメントに関するお問い合わせ、及び**osu!に関するヘルプが必要なもの**については[accounts@ppy.sh](mailto:accounts@ppy.sh)（アカウントクエリ）または、[osu@ppy.sh](mailto:osu@ppy.sh)（何でも）にメールを送信してください。
```

File: wiki/Do_you_really_want_to_ask_peppy/ja.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# 本当にpeppyに聞きたいですか？
+
+こんにちは。個人的に私（[peppy](https://osu.ppy.sh/users/2)）に連絡したいというあなたの衝動は理解していますが、これはあなたが私に助け/サポートを求る場所**ではない**ことを最初に理解するようにお願いします。私は個人的なメッセージとは別にosu!のサポートを継続して行っています。また、フォーラムのPMに回答するのは非常に難しいです。そのため、次のいずれかで連絡をしてください。
+
+## テクニカルサポート
+
+ほとんどの技術的な問題は、[ヘルプフォーラム](https://osu.ppy.sh/community/forums/5)で検索または投稿することで解決できます。問題を解決できる人が他にもたくさんいるので、私に連絡するよりも早く解決策を見つけることができるでしょう。
```
Comment (cl8n) [inline wiki/Do_you_really_want_to_ask_peppy/ja.md]:
```suggestion
ほとんどの技術的な問題は、[Helpフォーラム](https://osu.ppy.sh/community/forums/5)で検索または投稿することで解決できます。問題を解決できる人が他にもたくさんいるので、私に連絡するよりも早く解決策を見つけることができるでしょう。
```

File: wiki/Do_you_really_want_to_ask_peppy/ja.md (hunk 1 above)
Comment (ethernetcat) [reply wiki/Do_you_really_want_to_ask_peppy/ja.md]: Yes, I mistakenly bracketed😶
Commit the fix for it.

File: wiki/Do_you_really_want_to_ask_peppy/ja.md (hunk 2 above)
Comment (ethernetcat) [reply wiki/Do_you_really_want_to_ask_peppy/ja.md]: Fixed it.

**Comment by ethernetcat** [comment]
Thanks for the reviews and suggestions!
I committed the changes suggested in the review.

Thank you for taking the time for the review.

**Pull request merged**